budget/templates/layout.html

```diff
@@ -51,6 +51,7 @@ <h1 class="col-2"><a class="navbar-brand" href="{{ url_for(".home") }}">#! money
                     <a href="#" class="nav-link dropdown-toggle" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><strong>{{ g.project.name }}</strong> {{ _("options") }} <b class="caret"></b></a>
                     <ul class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
                         <li><a class="dropdown-item" href="{{ url_for(".edit_project") }}">{{ _("Project settings") }}</a></li>
+                        <li><a class="dropdown-item" href="{{ url_for(".export_project") }}">{{ _("Export project") }}</a></li>
```

Comment:
I'm not sure where this should be: what about having a section in the project's settings instead?